extras/indexing/src/main/java/org/apache/rya/indexing/IndexingFunctionRegistry.java

```diff
@@ -73,18 +73,22 @@
     
     
     public static Var getResultVarFromFunctionCall(URI function, List<ValueExpr> args) {
-        
         FUNCTION_TYPE type = SEARCH_FUNCTIONS.get(function);
-        
-        switch(type) {
-        case GEO: 
-            return findBinaryResultVar(args);
-        case FREETEXT:
-            return findLiteralResultVar(args);
-        case TEMPORAL:
-            return findBinaryResultVar(args);
-        default:
-            return null;
+        if (type == null) {
+        	return findBinaryResultVar(args);
+        }
+        else {
```

Comment:
This is kind of a style nit.

Since the conditional returns a value, you really don't need the else clause.